src/engine/shared/netban.h

```diff
@@ -229,11 +229,11 @@ void CNetBan::MakeBanInfo(const CBan<T> *pBan, char *pBuf, unsigned BuffSize, in
 	// add info part
 	if(pBan->m_Info.m_Expires != CBanInfo::EXPIRES_NEVER)
 	{
-		int Mins = ((pBan->m_Info.m_Expires - time_timestamp()) + 59) / 60;
+		int64_t Mins = ((pBan->m_Info.m_Expires - time_timestamp()) + 59) / 60;
```

Comment:
```suggestion
		int Mins = ((pBan->m_Info.m_Expires - time_timestamp()) + 59) / 60;
```
Since bans can only be 2 billion seconds long, this should be fine (and simpler).